dev/CommonStyles/Common_themeresources.xaml

```diff
@@ -17,6 +17,8 @@
             <Thickness x:Key="TextControlThemePadding">10,5,6,6</Thickness>
             <SolidColorBrush x:Key="ApplicationPageBackgroundThemeBrush" Color="{StaticResource SolidBackgroundFillColorBase}" />
             <StaticResource x:Key="DefaultTextForegroundThemeBrush" ResourceKey="TextFillColorPrimaryBrush" />
+            <SolidColorBrush x:Key="SystemControlFocusVisualPrimaryBrush" Color="{StaticResource FocusStrokeColorOuter}"/>
+            <SolidColorBrush x:Key="SystemControlFocusVisualSecondaryBrush" Color="{StaticResource FocusStrokeColorInner}"/>
```

Comment:
Would you like to incorporate those changes in one of the existing PRs (e.g. #4628) or should we create a new PR to fix this?

Reply:
Would you be able to add it to your open PR: 4635?

Sorry these comments are late. I couldn't figure out why nobody was responding to any of my PR comments and found out you need to publish your comments so others can see them. Confusing UI!

Reply:
No problem, I'll add it to #4635 :)
In case you didn't know there are actually two different ways to comment, one posts them immediately the other waits for them to be published as part of a review.